frontend/src/reducers/rootReducer.ts

```diff
@@ -45,6 +46,7 @@ export const reducerObject = {
   [reducerTypes.KEYCLOAK_READY]: keycloakReadySlice.reducer,
   [reducerTypes.MAP_VIEW_ZOOM]: mapViewZoomSlice.reducer,
   [reducerTypes.PROPERTY_NAMES]: propertyNameSlice.reducer,
+  [tenantsSlice.name]: tenantsSlice.reducer,
```

Comment:
Note that we do not need additional string constant files, we can reduce the complexity.